src/todobuddy/models/ToDo.java

```diff
@@ -0,0 +1,5 @@
+package todobuddy.models;
+
+public class ToDo {
```

Comment:
HuiYie, could you please help me rename this class to ToDoItem? using refactor function. thanks